ml-proto/host/format.mli

```diff
@@ -0,0 +1,7 @@
+open Sexpr
+
+val func_type : Types.func_type -> sexpr
+
+val expr : Kernel.expr -> sexpr
+val module_ : Kernel.module_ -> sexpr
```

Comment:
Yeah, I was struggling with a good name. I chose `Format` as the inverse to `Parse`, but later factored the actual printing to a separate module. Suggestions welcome.